zones/public/asia_east_1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "public/asia_east_1",
+  "pollerTimeout": 300,
+  "provider": "Rackspace",
+  "providerRegion": "HKG1",
+  "sourceIpAddresses": [
+    "127.0.0.1/27"
```

Comment:
Since all of these are using the ele collectors, it might be worth putting in the real ranges now.

Reply:
...which now makes me wonder if this particular content should live in a public repo. Maybe it's time to create a private, companion repo and configure the data-loader with a github auth token.

Reply:
The purpose of those addresses is so they can be accessed via the public api so customers know what ips need to be whitelisted.

I'm not sure any data-loader content that we're adding to prod would need to remain private?

Reply:
doh, yeah, in fact I'll probably find the existing list of IPs from some of our public docs

Reply:
...actually these IP addresses won't apply to the dev cluster. Remember that the branches in this repo map to data-loaders as:

Branch | Cluster
-------|---------
master | dev
staging | staging
production | production

I keep forgetting to mention. Going forward we should be using PRs and merge commits (not squashes) to move promote this content via the staging and production branches. I'll apply the collector public IP addresses during the production-branch PR.

Reply:
It might be worth talking about our method of achieving this without things getting messy.

The idea of another repo for dev was with the thinking that stage/prod would be the same and dev might have some different things in there... but this case devalues that.